index.bs

```diff
@@ -131,27 +138,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <h2 id="image-sequences">Image Sequences</h2>
 
-<p>An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
-<ul>
-<li>The [=Sequence Header OBUs=] SHALL have their <code>[=still_picture=]</code> flag set to 1 and SHOULD have their <code>[=reduced_still_picture_header=]</code> flags set to 1.
-</li>
-<li>The track handlers <code>vide</code> or <code>pict</code> MAY be used.
-</ul>
-
-<h2 id="alpha-images">Alpha Image Items and Sequences</h2>
-
+<p>
+    An <dfn>AV1 Image Sequence</dfn> is defined as a set of AV1 [=Temporal Units=] stored in an [=AV1 track=] as defined in [[!AV1-ISOBMFF]] with the following constraints:
+
+    - The [=AV1 Image Item=] should have its <code>[=still_picture=]</code> flag set to 1.
+    - The [=AV1 Image Item=] should have its <code>[=reduced_still_picture_header=]</code> flag set to 1.
+    - The track handler for an [=AV1 Image Sequence=] shall be <code>pict</code>.
+
+    <h2 id="alpha-images">Alpha Image Items and Sequences</h2>
 <p>An <dfn>AV1 Alpha Image Item</dfn> (respectively an <dfn>AV1 Alpha Image Sequence</dfn>) is an [=AV1 Image Item=] (respectively [=AV1 Image Sequence=]</dfn>) with the following additional constraints:
 <ul>
 <li>The <code>[=mono_chrome=]</code> field in the [=Sequence Header OBU=] SHALL be set to 1.</li>
 </ul>
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+
+  <h3 id="AVIF-general-brand">AVIF General Brand</h3>
+
+Files conformant with the brand-independent restrictions in this document shall include the brand
```

Comment:
We should be clearer about what the "brand-independent restrictions" are. Also, I think it should be phrased the other way around: if a file contains the 'avif' brand in the compatible_brands field, it shall be conformant to section 2, 3 or 4.

Reply:
I am OK with phrasing it the other way around, although I don't see what the big difference would be. One reason for the current structure is that I had only a single brand for images and images sequences. If I am swapping the phrasing around, it makes more sense if we have seprate brands for images and image sequences. 
Since we had already agreed to that we will have separate brands, I will just go ahead and defined the 'avis' brand for images sequences, and swap the phrasing around as you suggested.